language/functional_tests/tests/testsuite/block/block_prologue.mvir

```diff
@@ -3,9 +3,12 @@ import 0x0.LibraSystem;
 main() {
     // TODO: this will stop working once we make the Block prologue private. We should delete this
     // part (which will be done by the executor) and leave the second assert
-    LibraSystem.block_prologue(1);
+    LibraSystem.block_prologue(1, 2, h"aa", h"bb", 0xac);
 
-    assert(<ValidatorSet>rent_height() == 1, 77);
+    assert(LibraSystem.get_current_block_height() == 1, 77);
+    assert(LibraSystem.get_current_block_id() == h"aa", 77);
+    assert(LibraSystem.get_current_timestamp() == 2, 77);
+    assert(LibraSystem.get_current_proposer() == 0xac, 77);
     return;
 }
```

Comment:
We should add some additional tests to ensure that the height is increasing by 1, passing a bad time fails, passing a non-validator proposer fails, and so on.